Sources/StreamChat/Workers/Background/MessageSender.swift

```diff
@@ -117,6 +114,15 @@ class MessageSender: Worker {
             }
         }
     }
+    
+    func didUpdateConnectionState(_ state: WebSocketConnectionState) {
+        guard case WebSocketConnectionState.connected = state else { return }
```

Comment:
When the state is disconnected, dont we need to notify the message sender that it was disconnect?